packages/filefeeds/src/filefeeds.ts

```diff
@@ -158,20 +158,20 @@ export class OneSchemaFileFeedsClass extends EventEmitter {
         try {
           const resumeToken = window.localStorage.getItem(this.#resumeTokenKey)
           if (resumeToken) {
-            mergedParams.resumeToken = resumeToken
+            mergedParams.sessionToken = resumeToken
```

Comment:
I think we changed this and you need to revert your change here.

Reply:
Basically, we decided to—similar to the Importer SDK—the FF SDK sends the `resumeToken` value directly to the init endpoint and let that endpoint decide which ones of `sessionToken` and `resumeToken` to use.